docs/en/functionalities.rst

```diff
@@ -0,0 +1,413 @@
+- The User MUST have access to all issued Electronic Attestations by navigating the Catalog of the Wallet Instance. 
+
+The User MUST always be able to request the revocation of the Electronic Attestation of Attributes, even without the device where the Instance was activated. For more details, please refer to the `Management of Electronic Attestations`_ section. 
+
+In the event of communication issues between the systems of the Electronic Attestation of Attributes Provider and the Authentic Source, or if administrative or technical processes prevent the immediate issuance of the Electronic Attestation of Attributes, the request process MUST support deferred process. In this case:  
+- Upon reaching the final step of the process, the User MUST be presented with a message prompting them to wait until the Electronic Attestation of Attributes can be issued. 
+- The User MUST be notified by the Electronic Attestation of Attributes Provider once the Electronic Attestation of Attributes becomes available. 
+
+If the User encounters incorrect data in an already obtained or in-progress Electronic Attestation of Attributes, they SHOULD receive appropriate assistance. For more information, please refer to the `Assistance`_ section. 
+
+In case of errors, the User MUST receive clear, consistent messages that inform them and guide them toward resolution. For further details, please refer to the `Error Management`_ section. 
+
+If an Authentic Source (or an Electronic Attestation of Attributes Provider, should it coincide with the Authentic Source) intends to implement an Engagement Button to initiate the request process from their Touchpoint, they MUST ensure compliance with the graphical appearance and implementation requirements for the Engagement Button, as outlined in the `IT-Wallet System Brand Identity`_ section. 
+
+Layout of Electronic Attestations 
+==================================
+
+The Electronic Attestations obtained within the Wallet Instance SHOULD be displayed in a list within a Preview View. In this case, the Electronic Attestations MUST ensure a high level of recognizability and accessibility [REF_ACCESSIBILITY] of the information contained. Below are the requirements for displaying the Electronic Attestation that each Wallet Provider MUST adhere to in order to provide a consistent and accessible consultation and usage experience: 
+
+- The Electronic Attestation MUST be displayed correctly across all devices, ensuring a consistent experience on screens of varying sizes; 
+- The name of the Electronic Attestation MUST be clearly visible and always displayed in both the Detail View and the Preview View; 
+- The Electronic Attestation, in the Preview View, MUST display its validity status and MAY also include additional attributes to enhance the user experience and management; for example, it MAY display the name or logo of the Electronic Attestation of Attributes Provider or the PID Provider; 
```

Comment:
```suggestion
- The Electronic Attestation, in the Preview View, MUST display its validity status and MAY also include additional attributes to enhance the User experience and management; for example, it MAY display the name or logo of the Electronic Attestation of Attributes Provider or the PID Provider; 
```